src/shared/lib/getCanonicalMutationType.ts

```diff
@@ -1,9 +1,28 @@
 import {Mutation} from "../api/generated/CBioPortalAPI";
 
+type MISSENSE = "missense";
+type FRAME_SHIFT_INS = "frame_shift_ins";
+type FRAME_SHIFT_DEL = "frame_shift_del";
+type FRAMESHIFT = "frameshift";
+type NONSENSE = "nonsense";
+type SPLICE_SITE = "splice_site";
+type NONSTART = "nonstart";
+type NONSTOP = "nonstop";
+type IN_FRAME_DEL = "in_frame_del";
+type IN_FRAME_INS = "in_frame_ins";
+type INFRAME = "inframe";
+type TRUNCATING = "truncating";
+type FUSION = "fusion";
+type SILENT = "silent";
+type OTHER = "other";
+
 export type CanonicalMutationType =
-    "missense" | "frame_shift_ins" | "frame_shift_del" | "frameshift" |
-        "nonsense" | "splice_site" | "nonstart" | "nonstop" | "in_frame_del" |
-        "in_frame_ins" | "inframe" | "truncating" | "fusion" | "silent" | "other";
+    MISSENSE | FRAME_SHIFT_INS | FRAME_SHIFT_DEL | FRAMESHIFT |
+        NONSENSE | SPLICE_SITE | NONSTART | NONSTOP | IN_FRAME_DEL |
+        IN_FRAME_INS | INFRAME | TRUNCATING | FUSION | SILENT | OTHER;
+
+export type LollipopMutationType =
```

Comment:
I think this mutation type should have a different name. `Lollipop` is a visualization term, but these fewer types we choose are something conceptually separate from the lollipop. I don't really know what name fits best here. In legacy implementation, I called them `main` types, but that is not a good name either.